addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -363,7 +373,7 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
     updateCost(entry){
       entry.cost.errors = [];
 
-      if (!this.isPositiveInteger(Number(entry.cost.value))) {
+      if (!this.isPositiveInteger(entry.cost.value)) {
```

Comment:
Remove Number() as it will otherwise not show the error message when a field is left empty